packages/block-library/src/audio/edit.js

```diff
@@ -200,5 +206,13 @@ class AudioEdit extends Component {
 		/* eslint-enable jsx-a11y/no-static-element-interactions, jsx-a11y/onclick-has-role, jsx-a11y/click-events-have-key-events */
 	}
 }
-
-export default withNotices( AudioEdit );
+export default compose( [
+	withSelect( ( select ) => {
```

Comment:
What about `useSelect` ;)

Reply:
Hi @youknowriad I did not use useSelect because these are class components (where hooks could not be used), and most of the cases already had a withSelect usage. I guess if there are some benefits of the usage of the hook, we can create specific PR's with a refactor to hooks before the next block change.

Reply:
Makes sense